fed/mosaic-cell/src/main/java/org/eclipse/mosaic/fed/cell/utility/CapacityUtility.java

```diff
@@ -47,7 +45,7 @@ public final class CapacityUtility {
      * @throws IllegalArgumentException if delay or message size were set to invalid values
      */
     public static long calculateNeededCapacity(long messageSizeInBit, long delayInNs) throws IllegalArgumentException {
-        if (delayInNs <= 0 || messageSizeInBit <= 0) {
+        if (delayInNs <= 0 || messageSizeInBit < 0) {
```

Comment:
Why did you change this?

Reply:
See description in PR:

> When using EncodedPayload.EMPTY_PAYLOAD or V2xMessage.Empty to send dummy messages, e.g. for testing, it was found that mosaic-cell does not support message size of 0 bytes. However, I fixed this in the CapacityUtility so that such message would just consume no bandwidth and can therefore be simulated.

Reply:
Do we want to allow empty messages?

Reply:
We decided to revert this change. CELL again rejects empty messages. The test now sends an 8-byte message. We will evaluate in a follow-up issue how we can cope we empty payloads (e.g., by modelling additional header size added to all messages send by CELL or SNS)